client/src/pages/Home/Home.js

```diff
@@ -38,6 +38,13 @@ const Home = () => {
     setTimeout(() => {
       setLoading(false);
     }, 1000);
+
+    setTimeout(() => {
+      try {
+        // This is a hack for styling the mic
+        document.querySelector("holdable-button").shadowRoot.children[1].children[1].shadowRoot.children[1].setAttribute("style", "margin-left: -48px");
```

Comment:
@aymanjebril2 This was an ugly hack to fix that mic styling, but it seems to work